### ISSUE TITLE
[Suggestion/Mode] Reverse Traitor

### ISSUE THREAD

**Issue by Alblaka** [yogstation13/yogstation-classic#769]
Born from a random ideacrafting in OOC a moment ago:

--- REVERSE TRAITOR ---
X % of the crew (i.e. 20) start as 'Loyalists'. Loyalists can recognize each other by a special NT icon seperate to 'Loyality Implants', nor do Loyalists start implanted. Unless they are security/Captain, which start implanted, can only be Loyalists and will likely be the first to die.
Everyone else is a traitor with a 10TC uplink implant **BUT NO UPLINK** and the objective to kill all loyalists and then evacuate the station with X traitors aboard the shuttle (a third of those originally alive)
The loyalists are allowed to powergame (as well) and (attempt to) murder their way through the traitors, with the objective to have X loyalists (i.e. half of them) escape via shuttle.
There is no metashields or murderboning rules, except that Loyalists may not kill each other (unless in presence of traitors to prove 'they're truly aligned with the syndicate).

If the shuttle leaves with not enough traitors, but enough Loyalists, Loyalists win a Major Victory.
If the shuttle leaves with enough traitors and enough Loyalists, Loyalists score a Minor Victory.
If the shuttle leaves with enough traitors and not enough Loyalists, Syndicates score a Minor Victory.
If the shuttle leaves with not enough traitors, but no Loyalists, Syndicates score a Minor Victory.
If the shuttle leaves with enough traitors and no Loyalists, Syndicates score a Major Victory.

The gameplay would likely consist of the entire station beeing in between people running amok to figure out who has no traitor gear and everything going to shit because nobody does what his job is supposed to do, combined with loyalists trying to murder traitors in order to quickly gain gear to prove they are actually traitors. Oh, and obviously traitors will lynch each other trying to find the loyalists, too.

Maybe a bit VERY chaotic as a gamemode, and should be tried out as an event first.

**Comment by Kmc2000** [comment]
I like it, I can do this as an event when you're on alb , to test it out

**Comment by Alblaka** [comment]
Uh... Can I get someone else to bus that event? I still 'fondly' remember a certain Chrono Commando event...

Just kidding, sure, that would be lovely, thanks KMC ^^

**Comment by Kmc2000** [comment]
Lmao alb

Fuckin lmao

XD

**Comment by ghost** [comment]
everyone would buy minibombs and kill everything, not even fun

just

it would be rev, but with minibombs and eswords

**Comment by Alblaka** [comment]
Wouldnt exactly compare it to rev. Rev is about converting people and hunting a select known group.

This would be about hunting a larger, but entirely unknown group.

**Comment by SirCodey** [comment]
Have an announcement for people joining the server telling them this event is running, otherwise it's confusing as hell and several people died because of it.

**Comment by Alblaka** [comment]
Admin notice is in place, isnt it?

**Comment by harry2680** [comment]
What a clusterfuck his round is.

**Comment by ParagonPrime** [comment]
It's even more murderous than rev.

But somehow manages to be ridiculously fun nevertheless.

As long as there's a bit of admin intervention, making sure people aren't killing everyone not running around with an esword, and bringing in the MANY MANY MANY players who end up killed for the slightest shit back into the round- this fucking works.

But only with aforementioned oversight.

**Comment by Alblaka** [comment]
It's like revs with traitors instead of revs and adminbus everywhere. 
#TroubleInSyndicateStation

**Comment by hawkname** [comment]
It's pretty fun, better than I expected. But there needs to be a system worked out for late joiners, many whom join without being fully aware if they were loyalists or not. I'm not too sure, but apparently there were cases of loyalists cutting down each other because there was no R symbol on them.

**Comment by Kmc2000** [comment]
https://gyazo.com/7e041321e06a9193a7d2e11df8223f73

This would be a LOT better if it weren't a ghetto one man admin event, with proper IDS for the different sides, cos that was chaos

**Comment by LesVegetables** [comment]
its a terrible version of traitor deathmatch where the loyalists spawn at a huge disadvantage and the continuous spawning of players makes this shit never end

**Comment by ParagonPrime** [comment]
Vegetables, the loyalists were... actually doing surprisingly well. I was ghosted for a bit after I died and, if KMC hadn't brought a bunch of people back into the round as syndicate agents, they might very well have _won_. They were damn close to it in the end.

TL;DR-
1. Needs more coding and framework. A LOT more. Specific loyalist labels, automatic removal of loyalty implant from the tator-spawned Captain/etc (Cap nearly got killed at one point because everyone's so used to the "implant = NT" mentality), etcetera etcetera.
2. Needs bussing or SOME way to make sure people can get back into the round because the death rate is even higher than rev.

Hell, I might even advise setting a time limit on the round- **but allowing respawn** after a certain amount of time has passed since your death. And only as a character you haven't used before (so you get 3 respawns normally)

As a pure "once in a blue moon grief-fest event to get all the murderboner out of people's systems, with proper background work to keep it enjoyable even when you die within ten seconds"? I sure do like it. We can even cut rev out of rotation until we fix it with this, for that "ULTRAMURDERBONE" round so many people seem to advocate. Just... make sure the chance for it to be a Reverse Traitor round is really low.

**Comment by Alblaka** [comment]
>  if KMC hadn't brought a bunch of people back into the round as syndicate agents, they might very well have won. They were damn close to it in the end.

Those were ~3 dudes, from which one was a loyalist yet again. Technically a big buff to loyalists.
